storage/database/schema.go

```diff
@@ -105,6 +105,18 @@ type TxLookupEntry struct {
 	Index      uint64
 }
 
+// encodeBlockNumber encodes a block number as big endian uint64
+func encodeBlockNumber(number uint64) []byte {
+	enc := make([]byte, 8)
+	binary.BigEndian.PutUint64(enc, number)
+	return enc
+}
+
+// headerKeyPrefix = headerPrefix + num (uint64 big endian)
+func headerKeyPrefix(number uint64) []byte {
+	return append(headerPrefix, encodeBlockNumber(number)...)
```

Comment:
Why don't you use `common.Int64ToByteBigEndian(number)` as shown below?

Reply:
I'll reflect this change. Thanks.